.github/scripts/push-admin-console.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# When the builds succeeds, we need to place the results in the backend META-INF code and commit it.
+# Delete whatever is there and add the new build.
+
+TARGET="backend/src/main/resources/META-INF/resources/internal"
+
+rm -rf "${TARGET}"
+mkdir -p "${TARGET}"
+cp -r admin-console/src/main/webapp/build/* "${TARGET}"
+
+git config --global user.name "${GITHUB_ACTOR}[automatic action]"
```

Comment:
I'm not sure where this will be shown, but maybe the username should be a bit more specific:
```suggestion
git config --global user.name "${GITHUB_ACTOR}[admin-console push script]"
```

Reply:
This will show in the commit